### ISSUE TITLE
systemd should document ways of locking down `systemctl --host`

### ISSUE THREAD

**Issue by kakra** [systemd/systemd#13358]
**Is your feature request related to a problem? Please describe.**
We want to implement a management container that is sibling to other nspawn containers it manages. So far it was easy to get access to those containers using `systemctl -H parenthost/machine restart some.service`. We added an ssh-key so that machine can easily SSH to the host without requiring a password. But I wonder about security: I've found no documentation / clues about how to implement a wrapper or forced command on the SSH host side through `authorized_key`.

**Describe the solution you'd like**
systemd should come with an example wrapper or some documentation about how the SSH connection is actually used so admins can lock down such access with SSH forced commands.

**Comment by DemiMarie** [comment]
The OpenSSH documentation has everything you need for this, in `man 8 sshd`.  By disabling all forwarding, and forcing a specific command to be executed, it is possible to heavily lock-down key-based SSH.  Such options can be specific to an individual SSH key, and need not be system-wide.

**Comment by kakra** [comment]
Yes, sure... I know how to do it because I'm using it all the time to lock down access by key, e.g. for backup access. All that's needed would be an example of which command `systemctl -H ...` actually sends to the SSH host (for both variants `... -H host/machine` and `... -H host`).

Actually, most other software has an example for this, i.e. borgbackup has an example. They don't say `man 8 sshd` has everything you need.

> and forcing a specific command to be executed

Ah yes, here we are: Define "specific command".

Thanks. :-)

**Comment by boucman** [comment]
i'm pretty sure it's "systemd-stdio-bridge" which is used to connect it directly to dbus, but please check the source to make sure...

**Comment by kakra** [comment]
Okay, that's a starting point at least. I'll look into it and maybe create a patch for documentation. Thanks.

**Comment by doverride** [comment]
Yes something like this:
```
from="192.168.1.0/24",command="/usr/bin/systemd-stdio-bridge",no-port-forwarding,no-X11-forwarding,no-pty ssh-rsa ...
```

**Comment by srd424** [comment]
Looking at the source, I think it actually just sets up forwarding for appropriate unix domain sockets?

**Comment by kakra** [comment]
By coincidence, I found it's running `ssh -xT -- HOSTNAME systemd-stdio-bridge --machine=MACHINENAME`.

**Comment by melkamm12** [comment]
please sen tome gis projects every interested one via abiyuaxum@gmail.com

**Comment by foru007-del** [comment]
I can't understand what are you people saying 😥😥

**Comment by foru007-del** [comment]
Please tell me how could i learn this
